packages/devtools-reps/src/reps/grip.js

```diff
@@ -31,6 +31,7 @@ GripRep.propTypes = {
   onDOMNodeMouseOver: React.PropTypes.func,
   onDOMNodeMouseOut: React.PropTypes.func,
   onInspectIconClick: React.PropTypes.func,
+  noGrip: React.PropTypes.bool,
```

Comment:
The [Rep component](https://github.com/devtools-html/devtools-core/blob/master/packages/devtools-reps/src/reps/rep.js#L78) isn't really one, it's just a function that calls the correct rep, and thus has no propType.

We might type it with flow later, in which case we'll take care of adding this property into what can be in the `props` object.